extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -39,7 +39,7 @@ const publicAzureSettings: ProviderSettings = {
 			},
 			graphResource: {
 				id: SettingIds.graph,
-				endpoint: 'https://graph.windows.net',
+				endpoint: 'https://graph.windows.net/',
```

Comment:
where do we build the URL, is it in the library that we don't have control over? if so, this looks good to me, otherwise I think we should just check and append the slash if necessary when building the url.